features/create_child_topic.feature

```diff
@@ -0,0 +1,20 @@
+#language: fr
+
+Fonctionnalité: Créer des enfants
```

Comment:
Is that really the feature you want to explain step by step? 😲

Reply:
This feature allows user to create subtopic (child topic).